src/structures/Chat.js

```diff
@@ -165,29 +165,43 @@ class Chat extends Base {
      * Loads chat messages, sorted from earliest to latest.
      * @param {Object} searchOptions Options for searching messages. Right now only limit is supported.
      * @param {Number} [searchOptions.limit=50] The amount of messages to return. Note that the actual number of returned messages may be smaller if there aren't enough messages in the conversation. Set this to Infinity to load all messages.
+	 * @param {Number} [searchOptions.timestamp] Unix timestamp to return only messages after a given date/time. If you do not provide a valid timestamp, it will be ignored; and will use only searchOptions.limit value
      * @returns {Promise<Array<Message>>}
      */
     async fetchMessages(searchOptions) {
-        if (!searchOptions || !searchOptions.limit) {
-            searchOptions = { limit: 50 };
+        let {limit, timestamp} = searchOptions || {};
+
+        if(timestamp < 1) timestamp = null;
+        if(limit === undefined || limit === null) limit = (timestamp ? limit : 50);
+        if(timestamp){
+            const date = new Date(timestamp * 1000);
```

Comment:
Maybe passing in a js `Date` object instead of the unix timestamp would be easier for end users? What do you think?

Reply:
> Maybe passing in a js `Date` object instead of the unix timestamp would be easier for end users? What do you think?

For me this is not a problem, as long as it's valid. Whatever you find most interesting, it's fine with me!a